airbyte-integrations/connectors/source-twilio/Dockerfile

```diff
@@ -12,5 +12,5 @@ COPY main.py ./
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.1.16
+LABEL io.airbyte.version=0.1.17
```

Comment:
Adding a new stream is a minor change following the guidelines here: https://docs.airbyte.com/contributing-to-airbyte/#examples